kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/RequestStats.java

```diff
@@ -152,6 +157,12 @@ public class RequestStats {
 
     private final Map<ApiKeys, StatsLogger> apiKeysToStatsLogger = new ConcurrentHashMap<>();
 
+    private final Map<TopicPartition, StatsLogger> cachedLoggersForTopicPartitions = new ConcurrentHashMap<>();
```

Comment:
Can we keep the cache up to date by removing unloaded topics using `TopicOwnershipListener#whenUnload`?

Reply:
We definitely must clear it